spec/factories/projects.rb

```diff
@@ -7,6 +7,21 @@
     initialize_with { Tmuxinator::Project.new(file) }
   end
 
+  factory :project_with_force_attach, :class => Tmuxinator::Project do
+    transient do
+      file { YAML.load(File.read("#{File.expand_path("spec/fixtures/detach.yml")}")) }
+    end
+
+    initialize_with { Tmuxinator::Project.new(file, {:force_attach => true}) }
+  end
+
+  factory :project_with_force_detach, :class => Tmuxinator::Project do
+    transient do
+      file { YAML.load(File.read("#{File.expand_path("spec/fixtures/detach.yml")}")) }
+    end
+    initialize_with { Tmuxinator::Project.new(file, {:force_detach => true}) }
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.